boards/renesas/ek_ra8d1/ek_ra8d1.dts

```diff
@@ -160,3 +161,17 @@
 	pinctrl-0 = <&iic1_default>;
 	pinctrl-names = "default";
 };
+
+&sdhc1 {
+	compatible = "renesas,ra-sdhc";
+	interrupt-names = "accs", "card", "dma-req";
+	interrupts = <60 12>, <61 12>, <62 12>;
+	pinctrl-0 = <&sdhc1_default>;
+	pinctrl-names = "default";
+	status = "okay";
+	sdmmc {
+		compatible = "zephyr,sdmmc-disk";
+		disk-name = "SD";
+		status = "okay";
+	};
+};
```

Comment:
@thaoluonguw , @KhiemNguyenT , @soburi , the EK_RA8M1 and EK_RA8D1 doesn't support SD slot default on board but require external board, should we set its status as "okay" in board.dts?

Reply:
I moved the sdhc support with the status "okay" into app overlay for EK_RA8M1 and EK_RA8D1 because it requires external hardware and avoids some pins conflicts as above, keeps only for MCK-RA8T1
@KhiemNguyenT @thaoluonguw @soburi @duynguyenxa: please take a look. Thank you

Reply:
@danieldegrasse : sorry for inconvenience but could you please take another look? The driver implementation is unchanged, but the DTS support for the EK-RA8D1 and EK-RA8M1 boards has been updated as above discussed. Thank you

Reply:
So if I understand right, an external board is required for support here, which contains an SD card slot? A few questions:
- Is this external board standardized between multiple EVKs? That is, can it be reused on multiple Renesas EVKs?
- Is this board a product that can be purchased, or a custom rework to connect an SD card?

If the board is a product that can be purchased and is reusable across multiple boards, then I would suggest defining the board as a [shield](https://docs.zephyrproject.org/latest/hardware/porting/shields.html), so that it can be reused across multiple Renesas EVKs. You'll need to define a standard node name for the SDHC controller that exposes the pins the shield connects to- maybe something like `renesas_ext_sdhc`? See here for an example of this with NXP EVKs for an LCD interface: https://github.com/zephyrproject-rtos/zephyr/blob/2d60d248e846102867e4aca7a9e3d3e6ae591f06/boards/shields/lcd_par_s035/lcd_par_s035_8080.overlay#L33

Reply:
@danieldegrasse : 
> Is this external board standardized between multiple EVKs? That is, can it be reused on multiple Renesas EVKs?

Yes, it's standardized and can be reused for multiple Renesas EVKs.
> Is this board a product that can be purchased, or a custom rework to connect an SD card?

This is a product by digilent and can be purchased ([pmod sd](https://digilent.com/reference/pmod/pmodsd/start?srsltid=AfmBOopc03O7OGG2h70JZGR_v8o20X7MKZBe7GgHKNkWXjVIIfs3lW7W))
So, I defined a board shield for Pmod SD and overlay the pin connection for EK-RA8M1 and EK-RA8D1 inside. 
Please take a look. Thank you very much.